lib/upipe-framers/upipe_h264_framer.c

```diff
@@ -1365,7 +1365,7 @@ static void upipe_h264f_build_flow_def(struct upipe *upipe)
 
     if (upipe_h264f->duration) {
         UBASE_FATAL(upipe, uref_clock_set_latency(flow_def,
-                         upipe_h264f->input_latency + upipe_h264f->duration))
+                         upipe_h264f->input_latency + upipe_h264f->duration * 2))
```

Comment:
You're right.

I also see this code:

        case H264SEI_STRUCT_FRAME:
            UBASE_FATAL(upipe, uref_pic_set_progressive(uref))
            duration *= 2;
            break;

So no idea if this commit is correct. Also don't remember what it was trying to fix..